crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -71,6 +71,63 @@ impl Violation for Airflow3Removal {
     }
 }
 
+fn extract_name_from_slice(slice: &Expr) -> Option<String> {
+    match slice {
+        Expr::StringLiteral(ExprStringLiteral { value, .. }) => Some(value.to_string()),
+        _ => None,
+    }
+}
+
+pub(crate) fn removed_context_variable(checker: &mut Checker, expr: &Expr) {
+    const REMOVED_CONTEXT_KEYS: [&str; 12] = [
+        "conf",
+        "execution_date",
```

Comment:
For `execution_date` there is actually a replacement - in the docs: https://airflow.apache.org/docs/apache-airflow/stable/templates-ref.html#deprecated-variables - can you add this?

(Same for next_execution_date, prev_execution_date)

Reply:
We have renamed execution_date to logical_date at places but we have removed them as well:https://github.com/apache/airflow/pull/42404